CoffeeChat/Controller/Endpoints.swift

```diff
@@ -24,36 +24,70 @@ extension Endpoint {
         Endpoint.config.host = baseURL
         Endpoint.config.commonPath = "/api/v1"
     }
+    
```

Comment:
kinda strange thing but it looks like something might be wrong with the pbxproj? why is `Endpoints.swift` in `Controller` according to github?

Reply:
hmmm honestly im a little confused about our pbxproj file too?? not sure how to fix though.. Endpoints is in Networking for me locally but also the Networking folder is highlighted red..